### PR TITLE
[Enhance] Improve Swin Doc and Add Tabs enxtation

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Improve Swin Doc

## Modification

1. Add Tabs enxtation
image

2. folded the abstract

3. improve the readme of swin and swinv2.

## Use cases (Optional)
```
<!-- [TABS-BEGIN] -->

**Predict image**  # tab1

Content1  without "**xxxx**"

**Use the model**    # tab2

Content2  without "**xxxx**"

**Command**          # tab3

Content3 without "**xxxx**"

<!-- [TABS-END] -->
```

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: configs/swin_transformer/README.md [hunk 1]
```diff
@@ -48,6 +62,61 @@ The pre-trained models on ImageNet-21k are used to fine-tune, and therefore don'
...
+>>> out = model(inputs)
+>>> print(out.shape)
+torch.Size([1, 1000])
+>>> # To extract features.
+>>> outs = model.extract_feat(inputs)
+>>> print(outs[0].shape)
+torch.Size([1, 2048])
+```
+
+**Command**
```
Comment (tonysy) [inline configs/swin_transformer/README.md:101]: How about this:
```suggestion
**Train/Test Command**
```

Comment (Ezra-Yu) [reply configs/swin_transformer/README.md]: done

File: configs/swin_transformer/README.md [hunk 2]
```diff
@@ -48,6 +62,61 @@ The pre-trained models on ImageNet-21k are used to fine-tune, and therefore don'
...
+
+<!-- [TABS-BEGIN] -->
+
+**Predict image**
+
+```python
+>>> import torch
+>>> from mmcls.apis import init_model, inference_model
+>>>
+>>> model = init_model('configs/swin_transformer/swin-base_16xb64_in1k.py', 'https://download.openmmlab.com/mmclassification/v0/swin-transformer/convert/swin_tiny_patch4_window7_224-160bb0a5.pth')
```
Comment (yingfhu) [inline configs/swin_transformer/README.md:75]: base config does not match the tiny checkpoint

File: configs/swin_transformer/README.md [hunk 3]
```diff
@@ -48,6 +62,61 @@ The pre-trained models on ImageNet-21k are used to fine-tune, and therefore don'
...
+0.6649366617202759
+```
+
+**Use the model**
+
+```python
+>>> import torch
+>>> from mmcls.apis import init_model
+>>>
+>>> model = init_model('configs/swin_transformer/swin-base_16xb64_in1k.py', 'https://download.openmmlab.com/mmclassification/v0/swin-transformer/convert/swin_tiny_patch4_window7_224-160bb0a5.pth')
```
Comment (yingfhu) [inline configs/swin_transformer/README.md:89]: same and below

File: configs/swin_transformer/README.md [hunk 4]
```diff
@@ -48,6 +62,61 @@ The pre-trained models on ImageNet-21k are used to fine-tune, and therefore don'
...
+## How to use it?
+
+<!-- [TABS-BEGIN] -->
+
+**Predict image**
+
+```python
+>>> import torch
+>>> from mmcls.apis import init_model, inference_model
+>>>
```
Comment (yingfhu) [inline configs/swin_transformer/README.md:74]: extra `register_all_modules()` is needed to run the code

**Review by yingfhu — CHANGES_REQUESTED** [review]

File: configs/swin_transformer/README.md (hunk 2 above)
Comment (Ezra-Yu) [reply configs/swin_transformer/README.md]: done

File: configs/swin_transformer/README.md (hunk 4 above)
Comment (Ezra-Yu) [reply configs/swin_transformer/README.md:74]: done. add `register_all_modules()` in `init_model`

**Review by tonysy — APPROVED** [review]
LGTM

**Review by mzr1996 — APPROVED** [review]

**Pull request merged**